setup.cfg

```diff
@@ -2,7 +2,7 @@
 test=pytest
 
 [flake8]
-ignore = D100,D101,D102,D103,D104,D105,D106,D107,D202,E501,RST301,RST304,W503,E203,C417
+ignore = D100,D101,D102,D103,D104,D105,D106,D107,D202,E501,RST301,RST304,W503,E203,C417,T202
```

Comment:
t202 is for `print()` right? Is it needed?

Reply:
I cherry picked it from: #202 

@dmarjenburgh -> I copied this from your PR; as I ran into flake8 issues. Care to elaborate?

Reply:
T202 is:

> T202 Python 2.x reserved word print used.

Python 2.x is not relevant for this project, but flake8-print did trigger it. T202 has been removed in flake8-print 5.0.0 though, but we have `"flake8 < 5.0.0"` in `pyproj.toml`